web/src/components/Header/notificationInfo.ts

```diff
@@ -0,0 +1,385 @@
+          link: `${P2P_URL}/trades/${item.data.tradeId}`,
+          createdAt,
+        };
+      }
+      return {
+        text: t('newChatMessage', {
+          publicName: item.data.from,
+        }),
+        link: `${P2P_URL}/users/${item.data.from}`,
+        createdAt,
+      };
+
+    case 'checkCashed':
+      return {
+        text: t('checkCashed', item.data),
+        link: `${WALLETS_URL}/${item.data.cryptocurrency}`,
+        createdAt,
+      };
+    case 'moneyReceived': {
+      const nData = {
+        cryptocurrencycode: item.data.cryptocurrency.code,
+        cryptocurrencyamount: item.data.cryptocurrency.amount,
+        currencyamount: formatFiatWithDelimiters(item.data.currency.amount, 2),
+        currencycode: item.data.currency.code,
+        donor: item.data.donor,
+      };
+
+      return {
+        text: t('moneyReceived', nData),
+        link: `${WALLETS_URL}/${item.data.cryptocurrency.code}`,
+        createdAt,
+      };
+    }
+    case 'userTradeStatusChanged':
+      return {
+        text: t('tradePause'),
+        link: `${P2P_URL}/adverts/`,
+        createdAt,
+      };
+    case 'newReferral':
+      return {
+        text: t('newReferral', {
+          publicName: item.data.referral,
+        }),
+        link: `${PROFILE_URL}/referral`,
+        createdAt,
+      };
+    case 'dividendsReceived': {
+      const nData = {
+        cryptocurrencycode: item.data.cryptocurrency.code,
+        cryptocurrencyamount: item.data.cryptocurrency.amount,
+      };
+
+      return {
+        text: t('dividendsReceived', nData),
+        link: `${PROFILE_URL}/referral`,
+        createdAt,
+      };
+    }
+
+    case 'accountsMerged':
+      return {
+        text: t('Telegram and web accounts have been merged'),
+        link: `${PROFILE_URL}/telegram`,
+        createdAt,
+      };
+    case 'blockChainMoneyReceived':
+      return item.data.isDust
+        ? {
+            text: t('wallet-balance-loaded-dust', item.data),
+            link: `${WALLETS_URL}/${item.data.cryptocurrency.code}`,
+            createdAt,
+          }
+        : {
+            text: t('wallet-balance-loaded', item.data),
+            link: `${WALLETS_URL}/${item.data.cryptocurrency}`,
+            createdAt,
+          };
+    case 'blockChainMoneySent':
+      return {
+        text: t('payment-processed', item.data),
+        link: `${WALLETS_URL}/${item.data.cryptocurrency}`,
+        createdAt,
+      };
+    case 'verificationDecision':
+      if (item.data.status) {
+        return {
+          text: t('verification-confirmed'),
+          link: `${PROFILE_URL}`,
+          createdAt,
+        };
+      }
+      return {
+        text: t('verification-rejected'),
+        alert: t('verification-rejected-ext', { reason: item.data.reason }),
+        createdAt,
+      };
+
+    case 'mute':
+      return {
+        // eslint-disable-next-line radix
+        text: t('mute', { duration: parseInt(item.data.duration) }),
```

Comment:
```suggestion
        text: t('mute', { duration: parseInt(item.data.duration, 10) }),
```
и игнорирование eslint можно убрать